src/globus_sdk/services/transfer/data.py

```diff
@@ -197,13 +199,13 @@ def __init__(
 
         self["DATA"] = []
 
-        self.update(kwargs)
-        for option, value in kwargs.items():
-            logger.info(
-                "TransferData.{} = {} (option passed in via kwargs)".format(
-                    option, value
+        if additional_fields is not None:
+            self.update(additional_fields)
+            for option, value in additional_fields.items():
+                logger.info(
+                    "TransferData.{} = {} (option passed in via "
+                    "additional_fields)".format(option, value)
```

Comment:
The format only gets applied to the second string here. I think it would be good to switch to an f-string anyway.